tests/bin/Dockerfile

```diff
@@ -9,6 +11,7 @@ CMD echo "Installing tests..."
 
 ENV WP_TESTS_DIR=/tmp/wordpress-tests-lib
 ENV WP_CORE_DIR=/tmp/src/wordpress
+ENV WOO_VERSION=$WOO_INSTALL_VERSION
```

Comment:
This has to be set here because `install-wp-tests` is executed as a RUN command in the docker file and in that uses an intermediate container that will not use environment variables set in the `docker-compose.yml` file.